asciidoctorj-api/src/main/java/org/asciidoctor/Placement.java

```diff
@@ -2,12 +2,13 @@
 
 public enum Placement {
 
-    
-    TOP("top"), BOTTOM("bottom"), LEFT("left"), RIGHT("right");
-    
+
+    TOP("top"), BOTTOM("bottom"), LEFT("left"), RIGHT("right"),
```

Comment:
Maybe the new values is worth mentioning in the CHANGELOG.

Reply:
Yes, very good point.
Just realised that I left the CHANGELOG incomplete too :D
Thank you!